python-sdk/src/astro/settings.py

```diff
@@ -10,6 +10,8 @@
 SNOWFLAKE_SCHEMA = conf.get("astro_sdk", "snowflake_default_schema", fallback=SCHEMA)
 REDSHIFT_SCHEMA = conf.get("astro_sdk", "redshift_default_schema", fallback=SCHEMA)
 
+LOAD_FILE_ENABLE_NATIVE_FALLBACK = conf.get("astro_sdk", "load_file_enable_native_fallback", fallback=False)
```

Comment:
We should add this to https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/configurations.rst

Reply:
Added config docs 5b3f7e44e973799fe9508b4812aa695b9e8f91b8